tests/factory/from.ts

```diff
@@ -24,10 +22,10 @@ describe('xs.from', () => {
   });
 
   it('should convert a rejected promise to a stream', (done: any) => {
-    const stream = xs.from(Promise.reject('no'));
+    const stream = xs.from<string>(Promise.reject('no'));
```

Comment:
Is this change (and similar changes like this) necessary because strict features were turned on, or because the `src` changed? I'd like the previous test code to still pass, because that indicates no breaking changes for people who were using it as `xs.from(promise)` not `xs.from<string>(promise)`.

Reply:
Haha, actually this one no, I must have gone through the error list, it first errored later because x was unused line 30, I removed it, so there's no need for the type hint anymore =D.

Reply:
Went through the whole PR, this was the only case, where fixing one errors fixed 2.